src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import {IconAdd, IconDelete} from 'app/icons';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import SelectField from 'app/views/settings/components/forms/selectField';
+import TextField from 'app/views/settings/components/forms/textField';
+
+import {Category} from './utils';
+
+type Condition = {
+  category: Category;
+  match: string;
+};
+
+const categoryOptions = [
+  [Category.RELEASES, t('Releases')],
+  [Category.ENVIROMENTS, t('Enviroments')],
+  [Category.USERS, t('Users')],
+];
+
+type Props = {
+  conditions: Array<Condition>;
+  onAdd: () => void;
+  onDelete: (index: number) => () => void;
+  onChange: <T extends keyof Condition>(
+    index: number,
+    field: T,
+    value: Condition[T]
+  ) => void;
+};
+
+function ConditionFields({conditions, onAdd, onDelete, onChange}: Props) {
+  return (
+    <React.Fragment>
+      {conditions.map(({match, category}, index) => {
+        const displayDescription = index === 0;
+        return (
+          <div key={index}>
+            <Fields>
+              <SelectField
+                label={displayDescription && t('Category')}
+                help={displayDescription && t('This is a description')}
+                name="category"
+                choices={categoryOptions}
+                onChange={value => onChange(index, 'category', value)}
+                value={category}
+                inline={false}
+                hideControlState
+                showHelpInTooltip
+                required
+                stacked
+              />
+              <TextField
+                label={displayDescription && t('Match Conditions')}
+                help={displayDescription && t('This is a description')}
+                placeholder="ex. 1* or [I3].[0-9].*"
```

Comment:
Should this be translated?

Reply:
maybe only the `ex.`. I will do in a follow-up PR...thanks for pointing this out :)